File: benchmarks/array-1K/Main.java
public class Main {

    public static void main(String[] args) {
        long before = System.nanoTime();

        int[] arr = new int[1_000];
        for (int i = 0; i < 1_000; i++) {
            arr[i] = i;
        }

        long after = System.nanoTime();

        System.out.println(after - before);
    }

}

File: benchmarks/array-1K/main.zig
const std = @import("std");

pub fn main() !void {
    const before: i128 = std.time.nanoTimestamp();

    var arr: [1000]u32 = undefined;
    for (0..arr.len) |i| {
        arr[i] = @intCast(i);
    }

    const after: i128 = std.time.nanoTimestamp();

    std.debug.print("{}", .{after - before});
}